Revit_Engine/Create/Config/RevitPullConfig.cs

```diff
@@ -1,4 +1,26 @@
-﻿/*	
+/*
```

Comment:
Why doubled?

Reply:
By doubled I mean the whole copyright being doubled starting from line 2 (not shown in the preview here). Refers to the comments below too.

Reply:
Evidently the start of the copyright was not correct/what BHoMBot was expecting. I'll add you as an assignee, it'll be easiest for you to fix this for BHoMBot please :smile:

Reply:
I am the reviewer here, how can I be the assignee at the same time? 🤔

Reply:
Co-authoring of the PR is valid, and there are other reviewers who can then review your work that you've done in collaboration with BHoMBot :smile:

Reply:
Shouldn't the effort be spent on fixing BHoMBot instead of trying to push others to do the dirty job hm? I have just spent a chunk of my life scrolling through those changes, would not mind support in getting this done.

Reply:
BHoMBot is here to help us, not do everything for us. The dirty job has already been handled by CI/CD in raising the issue, instructing BHoMBot to do what it can with the copyrights, and add copyright headers to files deemed not to have compliance copyright at all (not just an incorrect year). You've identified 3 files where there's duplication caused by incompliant files, it's far easier for you to resolve this having identified them than it is for BHoMBot to try and resolve them, given that BHoMBot is not a sentient being, but a bot instructed to do things by CI/CD. Ultimately, if you demand for BHoMBot to make the changes, you are simply pushing the job back onto me, and I'm trying to look after 56 PRs to your 1 to 5. Given that BHoMBot changed approximately 6,000 files, indicating a 0.05% of problems from the 3 files you've found here, I'd say it was pretty good :wink:

Reply:
That and co-authoring PRs is encouraged, so you can start as a reviewer and be promoted to assignee to collaboratively resolve and issue :smile: